From 56aef354cb75892c4db6e71be10937bc8965eb5b Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Wed, 9 Jun 2010 03:56:08 +0200
Subject: [PATCH 2/2] build: Disable GFDL info files and useless man pages

We do not install the info file due to GFDL, and because it would
require an inetutils-doc package. Nor the man pages from upstream
generated with help2man as they are problematic for cross-building
and contain no additional information to what is already available
via --help output.

Instead we ship our own proper man pages.

Not forwarded upstream due to GNU policies regarding man pages.
---
 Makefile.am  | 1 -
 configure.ac | 3 ---
 2 files changed, 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 5df19417..127e285c 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -25,7 +25,6 @@ EXTRA_DIST = paths ChangeLog.0 summary.sh.in CHECKLIST
 SUBDIRS = lib \
 	libinetutils libtelnet libicmp libls \
 	src telnet telnetd ftp ftpd talk talkd whois ping ifconfig \
-	doc man \
 	tests
 
 DISTCLEANFILES = pathdefs.make paths.defs \
diff --git a/configure.ac b/configure.ac
index 86136fb3..f954fb3a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -143,7 +143,6 @@ AC_PROG_RANLIB
 AC_PROG_YACC
 AC_PROG_LN_S
 AC_PROG_SED
-AM_MISSING_PROG(HELP2MAN, help2man, $missing_dir)
 AC_ARG_VAR(GREP, [Location of preferred 'grep' utility.])
 AC_ARG_VAR(EGREP, [Location of preferred 'egrep' utility.])
 AC_ARG_VAR(FGREP, [Location of preferred 'fgrep' utility.])
@@ -1073,8 +1072,6 @@ whois/Makefile
 ping/Makefile
 ifconfig/Makefile
 ifconfig/system/Makefile
-doc/Makefile
-man/Makefile
 tests/Makefile
 confpaths.h:confpaths.h.in
 ])
-- 
2.30.0.284.gd98b1dd5eaa7

